### PR TITLE
mindforger: 1.48.2 -> 1.52.0

### PR DESCRIPTION
**Pull request by cyplo**
###### Motivation for this change
- new upstream version

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
  - tested under i3:
    - fresh app start
    - creating new repository with default content
    - opening an existing repository
    - restart (app opens up previously opened repository by default)
    - navigation using keyboard shortcuts
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cyplo** [comment]
@s9gf4ult if you could take a look that would be grand

**Review by s9gf4ult — APPROVED** [review]

**Comment by cyplo** [comment]
@Staberinde could you try testing on yours as well, I can see you got excited by the PR :)

**Comment by Staberinde** [comment]
Haha, I'll see what I can do

File: pkgs/applications/editors/mindforger/default.nix [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
   };
 
-  nativeBuildInputs = [ qmake ] ;
-  buildInputs = [ qtbase qtwebkit ] ;
+  nativeBuildInputs = [ qmake wrapGAppsHook ] ;
+  buildInputs = [ qtbase qtwebkit cmark-gfm ] ;
 
   doCheck = true;
 
   enableParallelBuilding = true ;
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/mindforger/default.nix:25]:
```suggestion
```

Automatically set by qmake.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by cyplo** [comment]
thanks @SuperSandro2000 , if you could take another look :)

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 106741` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mindforger</li>
  </ul>
</details>

File: pkgs/applications/editors/mindforger/default.nix [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
 
-  nativeBuildInputs = [ qmake ] ;
-  buildInputs = [ qtbase qtwebkit ] ;
+  nativeBuildInputs = [ qmake wrapGAppsHook ] ;
+  buildInputs = [ qtbase qtwebkit cmark-gfm ] ;
 
   doCheck = true;
 
-  enableParallelBuilding = true ;
+  prePatch= ''
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/mindforger/default.nix:25]: Can we move this into postPatch? Otherwise some upstream patches might not apply properly.

**Comment by SuperSandro2000** [comment]
Building on darwin is fixed by #107898

File: pkgs/applications/editors/mindforger/default.nix (hunk 2 above)
Comment (cyplo) [reply pkgs/applications/editors/mindforger/default.nix:25]: let me try

Comment (cyplo) [reply pkgs/applications/editors/mindforger/default.nix]: yeah well spotted, should be fixed now, I retested the GUI manually and all seems fine

**Comment by cyplo** [comment]
@SuperSandro2000 if you could take another look, seems fine now on my end at least :) thanks !

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**